pkg/apiserver/etcd_config.go

```diff
@@ -115,13 +116,13 @@ func (c completedEtcdConfig) NewServer(stopCh <-chan struct{}) (*ServiceCatalogA
 			// we've sucessfully installed, so hook the stopCh to the destroy func of all the sucessfully installed apigroups
 			for _, mappings := range groupInfo.VersionedResourcesStorageMap { // gv to resource mappings
 				for _, storage := range mappings { // resource name (brokers, brokers/status) to backing storage
-					go func() {
```

Comment:
thoughts on a scoping variable rather than a function parameter?

```
storage := storage
```

Or a comment, since this is such an easy mistake to make? I don't want someone seeing it later and thinking they're optimizing something.

What did the golint output look like? How did we miss that in the build?

Reply:
I don't know how it was missed, but the only way to ensure the variable is correct at execution is by passing it as a parameter.

Reply:
Actually I take that back, you could copy it as you said. But I think it's clearer with the parameter personally.

Reply:
ya, no need to hold this up. we could bike shed on it for hours.

Reply:
This go routine semantics of lazy binding in Golang hits again.